packages/core/src/controllers/ConnectorController.ts

```diff
@@ -87,6 +107,9 @@ export const ConnectorController = {
         themeVariables,
         w3mThemeVariables: getW3mThemeVariables(themeVariables, themeMode)
       })
+      this.setConnectors([authConnector])
```

Comment:
Why set a aingle connector? Shouldnt you append/replace?

Reply:
You mean why **not**? The `setConnectors` business logic refactored a bit and designed to handle multiple connectors at once. For these kind of specific single setters, just replacing them by passing connector in an array.

Reply:
Big refactors will be introduced to the adapters (wagmi/ethers/ethers5) soon, so we could move this kind of improvements to there.